android-sdk/tasks/main.yml

```diff
@@ -75,12 +75,33 @@
     dest: /tmp/{{ config_file }}
     force: yes
 
-# install the config file
+- name: Get oc version
+  shell: "oc version | grep openshift | awk '{print $2}' | awk '{print substr($0,4,1)}'"
+  register: version_output
+
+- set_fact:
+    openshift_minor_version: "{{ version_output.stdout }}"
+  failed_when: version_output is undefined
+
+# install the config file - OSCP 3.5 and greater
 -
   name: "Copy the Android SDK config file to the container"
   command: "oc cp --namespace={{ project_name }} /tmp/{{ config_file }} {{ android_sdk_podname }}:/opt/tools/{{ config_file }}"
   register: cp_cmd
-  failed_when: cp_cmd.stdout != ""
+  when: openshift_minor_version not in ["3","4"]
+
+- debug: var=cp_cmd
+
+# install the config file - OSCP 3.4 and lower
+-
+  name: "Copy the Android SDK config file to the container"
+  shell: "oc exec -i {{ android_sdk_podname }} --namespace={{ project_name }} -- bash -c 'cat > /opt/tools/{{ config_file }}' < /tmp/{{ config_file }}"
+  when: openshift_minor_version in ["3","4"]
+
+-
+  name: "Set permission on the file"
+  shell: "oc exec -i {{ android_sdk_podname }} --namespace={{ project_name }} -- bash -c 'chmod 775 /opt/tools/{{ config_file }}'"
+  when: openshift_minor_version in ["3", "4"]
```

Comment:
Since `oc exec` is available in both release, why we don't just use the same command? If we just use `oc exec`, it means we don't need to check the version of openshift, which less likely to introduce a bug?